packages/react/src/experimental/OneDataCollection/visualizations/collection/Table/components/Row.tsx

```diff
@@ -94,8 +97,55 @@ const RowComponentInner = <
 
   const itemActions = filterItemActions(source.itemActions, item)
 
+  // gets the primary item actions (max 2)
+  const primaryItemActions = itemActions
```

Comment:
As the actions logic is the same as in the List view we should extract it and the render of the actions to a component used by the table and the view